src/components/pages/Dashboard/parseObjectContentToString.js

```diff
@@ -0,0 +1,12 @@
+const parseObjectContentToString = obj => {
+  return obj.map(farmer => ({
```

Comment:
So looks like this new file has the same function as 'cleanFarmersData' in farmerHandlers. But cleanFarmersData is still the one in use? It is used in App.js, line #54. The idea of having it in the first place was to only perform the filtering and conversion once when new data arrives. If it's moved to the Dashboard to use parseObjectContentToString.js, this will be performed everytime that the Dashboard loads (a lot of times).

So if this is not going to be used, I'd removed it completely. And if this is going to be used, there is the concern of if this should happen in the Dashboard as I just said. If it happens in App, then this shouldn't be inside of Dashboard.

Anyway I believe this was not the best place to make this adjustment, it's very out of scope for 'reset user password'.